packit/cli/builds/copr_build.py

```diff
@@ -105,9 +110,17 @@
     default=False,
     is_flag=True,
 )
+@click.option(
+    "--packages",
+    help=(
+        "Comma separated list of packages to be built. "
+        "(defaults to all the packages listed inside the config)"
+    ),
+)
 @click.argument("path_or_url", type=LocalProjectParameter(), default=os.path.curdir)
 @pass_config
 @cover_packit_exception
+@iterate_packages
```

Comment:
The description is slightly different from command to command (it is the same just for the build commands...) and I saw that also all the other options are duplicated, so I think it will feel weird to look for this one elsewhere.